common/persistence/sql/sqlplugin/postgresql/visibility.go

```diff
@@ -152,12 +152,12 @@ func (pdb *db) ReplaceIntoVisibility(row *sqlplugin.VisibilityRow) (sql.Result,
 }
 
 // DeleteFromVisibility deletes a row from visibility table if it exist
-func (pdb *db) DeleteFromVisibility(filter *sqlplugin.VisibilityFilter) (sql.Result, error) {
+func (pdb *db) DeleteFromVisibility(filter sqlplugin.VisibilityDeleteFilter) (sql.Result, error) {
```

Comment:
Why do you remove pointer?

Reply:
less GC, since the filter type is small enough